packages/graphiql/package.json

```diff
@@ -37,15 +42,12 @@
       "git add"
     ]
   },
-  "browserify": {
-    "transform": [
-      "babelify"
-    ]
-  },
   "dependencies": {
     "codemirror": "^5.47.0",
     "codemirror-graphql": "^0.11.2",
     "copy-to-clipboard": "^3.2.0",
+    "graphql": "14.5.6",
+    "isomorphic-fetch": "^2.2.1",
```

Comment:
We should try to minimise dependencies; I really don't think we should add `isomorphic-fetch`.

We shouldn't pin a specific version of `graphql` as this will massively increase the likelihood of users having multiple versions of GraphQL in their `node_modules` folder. If possible, I don't think we should specify `graphql` at all (we didn't previously?). If necessary, add it as a `devDependency` instead?

Reply:
@benjie here we do load isomorphic fetch asynchronously, only in `renderGraphiQL`, so it doesn't get bundled at all with the default GraphiQL entrypoint, and as an optional chunk if they load the `render.min.js` entrypoint.

@benjie good point, i think we should make `graphql` only a peer, dev and/or optional dependency of all the modules, with ranges for peers/optional like we do already, and then bundle it with the CDN bundles.  I ended up forcing it everywhere because i found with 14.5.8 it was clashing somehow with something that required 14.5.6? need to get to the bottom of that.

Reply:
Why do we need `isomorphic-fetch` when we didn't in the past? Can't this be something the user adds if they need it rather than us polyfilling for them? I'm keen to avoid additional dependencies for the npm package whether or not they're bundle split/asynchronously loaded unless we absolutely need them. `node_modules` is already deep enough :wink:

Reply:
because of the purpose of `renderGraphiQL`.

with `graphiql.min.js` its bring your own fetch implementation for sure
but with `renderGraphiQL`, its meant to be the most minimal lines to get to a working implementation, with everything taken care of, including the DOM render and all. see the new examples in the netlify build (/render). a graphiql instance in 10 lines basically, plus that wrapper CSS